pdcupdater/handlers/depchain/base.py

```diff
@@ -20,6 +20,16 @@ class BaseKojiDepChainHandler(pdcupdater.handlers.BaseHandler):
     parent_type = None
     child_type = None
 
+    def construct_topic(self, config):
+        # Return a single hardcoded topic when using STOMP
+        if config.get('stomp_uri'):
```

Comment:
@ralphbean Do you think it'd be a good idea if I checked if ZMQ was disabled as well?